src/aiy/cloudspeech.py

```diff
@@ -47,7 +47,9 @@ def recognize(self):
         self._request.set_endpointer_cb(self._endpointer_callback)
         self._recorder.add_processor(self._request)
         text = self._request.do_request().transcript
-        if self._hotwords and text:
```

Comment:
Please add documentation for `immediate` above, eg:
```
Args:
    immediate: ignore the hotword list, even if it has been populated
```

Reply:
Should I place the example in the function documentation too?

Reply:
Yep, hopefully people who need it can find it there :)